practicesharp/Equation_Validator.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace basic_calculator
+{
+    class Equation_Validator
+    {
+        public string equation_s { get; private set; }
```

Comment:
Equation validator should just check if an equation is valid, is should not have a state.
Also think that same instance of this class may be used in different places (Which use the same validation).